src/goose/toolkit/developer.py

```diff
@@ -34,10 +34,10 @@ class Developer(Toolkit):
 
     def system(self) -> str:
         """Retrieve system configuration details for developer"""
-        hints_path = Path('.goosehints')
+        hints_path = Path(".goosehints")
         system_prompt = Message.load("prompts/developer.jinja").text
         if hints_path.is_file():
-            goosehints = hints_path.read_text()
+            goosehints = render_template(hints_path)
```

Comment:
i found the `.goose.jinja` file in this repo but couldn't find a reference on where/how it was being used
https://github.com/squareup/goose/blob/cffd4bfe8eaac603ec84c4c3b9e7f204cc91a948/.goose.jinja#L1